.github/workflows/build.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
           restore-keys: cypress-${{ runner.os }}-
       - name: Authenticate with private NPM package
         run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+        run: echo "@contentful:registry=https://npm.pkg.github.com" >> ~/.npmrc
+        run: echo "//npm.pkg.github.com/:_authToken=${{secrets.GH_TOKEN}}" >> ~/.npmrc        
```

Comment:
This whole setup should not be necessary. This is a public repository that should only use public dependencies. Public packages would therefore be published to npmjs. Similar to our actual users, we should only fetch our node modules via npmjs to not mix up things in here.

Please be mindful to manually turn the package to "public" in the npmjs registry though :)

Reply:
Sounds good - I'm going to make sure we publish it under the correct name as when you look it up it doesn't exist https://github.com/orgs/contentful/packages?tab=packages&q=experiences. So I'll have to rename it

Reply:
Note that we would still push this to github thanks to the `.npmrc` in this repo :) But this config here is only about installing modules which should still happen via npm

Reply:
Btw, so many packages with `experience` ^^ https://github.com/orgs/contentful/packages?q=experiences&tab=packages&q=experience